skore/src/skore/sklearn/_comparison/report.py

```diff
@@ -287,7 +289,8 @@ def get_predictions(
             - "train" : use the train set provided when creating the report.
             - "X_y" : use the provided `X` and `y` to compute the metric.
 
-        response_method : {"predict", "predict_proba", "decision_function"}
+        response_method : {"predict", "predict_proba", "decision_function"},
+        default="predict"
```

Comment:
```suggestion
        response_method : {"predict", "predict_proba", "decision_function"},
                          default="predict"
```
Sphinx needs this indenting. Same below.